template/pyClient.js

```diff
@@ -83,44 +87,65 @@ function getQueryParamBlock(queryMap) {
   `;
 }
 
-function getServiceUrlBlock (isSecure,isBasicAuth,urlProtocol,urlHost,urlPath) {
```

Comment:
instead of one param for each auth mode, let's use a single param and define constants for different authmode